script/config-center/apollo/apollo-config.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#  Copyright 2001-2006 The Apache Software Foundation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ----------------------------------------------------------------------------
+#
+#   Copyright (c) 2001-2006 The Apache Software Foundation.  All rights
+#   reserved.
+
+# apollo open api, click on the link for details:
+# https://github.com/ctripcorp/apollo/wiki/Apollo%E5%BC%80%E6%94%BE%E5%B9%B3%E5%8F%B0
+
+# add config: http://{portal_address}/openapi/v1/envs/{env}/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/items
+# publish config: http://{portal_address}/openapi/v1/envs/{env}/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/releases
+
+
+for line in $(cat apollo-params.txt); do
+	key=${line%%=*}
+	value=${line#*=}
+	case ${key} in
+	portalAddr)
+		portalAddr=${value}
+		;;
+	env)
+		env=${value}
+		;;
+	appId)
+		appId=${value}
+		;;
+	clusterName)
+		clusterName=${value}
+		;;
+	namespaceName)
+		namespaceName=${value}
+		;;
+	dataChangeCreatedBy)
+		dataChangeCreatedBy=${value}
+		;;
+	releasedBy)
+		releasedBy=${value}
+		;;
+	token)
+		token=${value}
+		;;
+	*)
+		echo "invalid param"
```

Comment:
This could be more clearly to point to config file.

Reply:
> This could be more clearly to point to config file.

invalid param，please refer to apollo-params.txt
may i?